packages/search/src/util/pagination/paginator.ts

```diff
@@ -0,0 +1,5 @@
+export interface IPageResponse {
+  cursor: string;
+  total: number;
```

Comment:
is this total records or pages?

Reply:
This is total records and I've added documentation to clarify that.